### ISSUE TITLE
duplicated rows in Azure DevOps output

### ISSUE THREAD

**Issue by fmms** [otykier/TabularEditor#329]
Hi,

i have a cosmetic issue in the Azure DevOps output:
![image](https://user-images.githubusercontent.com/908734/65357571-a7323000-dbf7-11e9-9e5a-04c528a30179.png)

Why is every line except the information about the dependency tree shown twice?

I have the following step in my multi-stage YAML based pipeline:
```
  - task: PowerShell@2
    displayName: Deploying AS Cube
    inputs:
      targetType: 'inline'
      script: |
        $server = "asazure://westeurope.asazure.windows.net/$(aaServerName):rw"
        Write-Host "Deploying to $server"
        $p = Start-Process -FilePath "C:\Program Files (x86)\TabularEditor\TabularEditor.exe" -NoNewWindow -PassThru -Wait -ArgumentList "$(Agent.BuildDirectory)/beye/as/Model.asdatabase -VSTS -DEPLOY $server database -LOGIN $(asAdminUser) $(asAdminPassword) -OVERWRITE -CONNECTIONS -ROLES"
        Write-Host "finished executing process"
        exit $p.ExitCode
```

Thanks alot

**Comment by otykier** [comment]
This issue ties back to the fact that TabularEditor.exe is not a console application, see also #291 . To make sure that we get any output at all, regardless of how the executable is invoked, it was necessary to write output to both the StandardOutput stream, as well as the parent console window.

When executing TabularEditor.exe through a normal Command Line task in Azure DevOps, we only get one of the streams in the output (and so no duplicated rows), but for some reason, the `Start-Process` PowerShell command captures both streams.

I'm not a PowerShell expert, but I think you might be able to get rid of the StandardOutput stream, by [specifying `-RedirectStandardOutput` in the call to `Start-Process`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/start-process?view=powershell-6#parameters). By sending one stream to a dummy file, you should end up with only one set of rows in the Pipeline log...

Hope this helps.

**Comment by fmms** [comment]
Hi @otykier , 

thanks for the quick response. However, so far nothing i have tried works.
- if I add ```-RedirectStandardOutput NUL ``` then I do not get any output at all anymore.
- if I remove ```-NoNewWindow``` I do not get any output either.

So far I did not find any solution to separate the streams.

It is not urgent but right now looks a bit strange ...

**Comment by fmms** [comment]
The  same behaviour has in fact been described in #154 and #241 .

**Comment by otykier** [comment]
Fixed in [2.8.6](https://github.com/otykier/TabularEditor/releases/tag/2.8.6).

**Comment by fmms** [comment]
Thanks a lot. I can confirm this to be fixed as well:
![image](https://user-images.githubusercontent.com/908734/66894497-d66a7000-eff0-11e9-8161-663d51566f84.png)

Thanks @albertospelta for your PR.
